examples/example-kotlin/README.md

```diff
@@ -15,7 +15,7 @@ To build the examples,
 
 2. Run in this directory:
 ```
-$ ./gradlew installDist
+$ ../gradlew installDist
```

Comment:
Why `../gradlew` here and `../../gradlew` in examples/android/README.md?

Reply:
Because in android README, the gradlew command was following another cd command.